packages/unified-pipeline/src/components/Canvas/utils/LROrientation/Constants.ts

```diff
@@ -17,7 +17,7 @@ export const GROUP_NODE_VERTICAL_ALIGNMENT_MARGIN = NODE_VERTICAL_MARGIN
 
 /* Step node constants */
 export const STEP_NODE_WIDTH = 140
-export const STEP_NODE_HEIGHT = 80
+export const STEP_NODE_HEIGHT = 60
```

Comment:
As per figma [designs](https://www.figma.com/design/Lu5QJyq9tQEjHMeQmQhbAd/Gitness-Product-Design---Stage-2?node-id=5318-292477&node-type=FRAME&t=0o1eHvXUwfEjPPts-0), height should be `80px`.

Also, `60px` is not able to accommodate entire content.
<img width="341" alt="image" src="https://github.com/user-attachments/assets/52f95540-3c2b-4a97-aa9a-7ca265b15290">